### ISSUE TITLE
Error in plugin 'sass'

### ISSUE THREAD

**Issue by olzhaskaliyev** [yeoman/generator-webapp#685]
i just added style file and import them in main.scss

_articles.scss
```
.article-card {
  height: 230px;
}
```
main.scss
```
...
@import "articles";
...
```
and got this error on `gulp serve` or another command like `gulp build`
```
"C:\Program Files\JetBrains\WebStorm 2017.2\bin\runnerw.exe" "C:\Program Files\nodejs\node.exe" C:\Job\yeoman-webapp\node_modules\gulp\bin\gulp.js --color --gulpfile C:\Job\yeoman-webapp\gulpfile.js serve
[18:35:41] Using gulpfile C:\Job\yeoman-webapp\gulpfile.js
[18:35:41] Starting 'serve'...
[18:35:41] Starting 'clean'...
[18:35:41] Starting 'wiredep'...
[18:35:41] Finished 'wiredep' after 23 ms
[18:35:41] Finished 'serve' after 34 ms
[18:35:41] Finished 'clean' after 120 ms
[18:35:41] Starting 'styles'...
[18:35:42] Starting 'scripts'...
[18:35:43] Starting 'fonts'...
[18:35:43] Finished 'fonts' after 48 ms
[Error in plugin 'sass18:35:43] Finished ''
stylesMessage:
' after     1.49 sapp\styles\main.scss
Error: File to import not found or unreadable: articles.
       Parent style sheet: C:/Job/yeoman-webapp/app/styles/main.scss
        on line 5 of app/styles/main.scss
>> @import "index", "articles";
   ^


[18:35:43] Finished 'scripts' after 1.39 s
[Browsersync] Access URLs:
 --------------------------------------
       Local: http://localhost:9000
    External: http://192.168.1.151:9000
 --------------------------------------
          UI: http://localhost:3001
 UI External: http://192.168.1.151:3001
 --------------------------------------
[Browsersync] Serving files from: .tmp
[Browsersync] Serving files from: app

```
pure install, nothing changed except main.scss, all how in "Getting start"
p.s.: it's problem very old, but now i want fixed that, with you.
p.p.s.: Sorry for my english. thx for future answer

**Comment by neilhem** [comment]
Hi, `_articles.scss` file is right in `app/styles` folder?

**Comment by olzhaskaliyev** [comment]
yes, it is! Right in same place, where `main.scss`

**Comment by olzhaskaliyev** [comment]
Forgot to mention - my platform is Windows 10 Pro x64, Node.js 6.11.1 (LTS)

**Comment by neilhem** [comment]
Using generator-webapp 3.0.1, no problem with importing sass files.

**Comment by olzhaskaliyev** [comment]
i too used generator-webapp 3.0.1
i saw that problem in web, but is no solutions. i'll write more details, when come to pc (I think we have a time difference of about 9 hours).
Thanks for quick answers )

**Comment by olzhaskaliyev** [comment]
Wow! You are from KZ like me!

**Comment by neilhem** [comment]
Have you tried just using single file imports? E.g.
```scss
@import "index";
@import "articles";
```
Looks like you are using IDE, so maybe duplicate of https://github.com/sass/node-sass/issues/1894

**Comment by olzhaskaliyev** [comment]
wow, it's look like solution, i'll try tomorrow, and if it will help, issues can to be close.

**Comment by olzhaskaliyev** [comment]
Thanks for answers, but safe write not help to me, and gulp-wait it's imprecise hack i think.
It's realy looks like files is lock when clean and wiredep tasks run in parallel with [runSequence](https://www.npmjs.com/package/run-sequence). It's often happens when project is big. I tried to extract this tasks from array, and it's helped. But i don't know - is it correct?

**Comment by neilhem** [comment]
I have never had such problem, try to find answer in [stackoverflow](https://stackoverflow.com/). Closing issue as not a bug with generator itself.

**Comment by unrealhero18** [comment]
https://garystanton.co.uk/gulp-sass-file-to-import-not-found-or-unreadable/
